python/brainyflow.py

```
@@ -17,78 +17,63 @@
-    
+        _local = self._local
+        class LocalProxy:
+            def __getattr__(self, key: str) -> Any: return _get_from_stores(key, _local, Error=AttributeError)
+            def __getitem__(self, key: str) -> Any: return _get_from_stores(key, _local)
+            def __setattr__(self, key: str, value: Any) -> None: _local[key] = value
+            def __setitem__(self, key: str, value: Any) -> None: _local[key] = value
+            def __delattr__(self, key: str) -> None: _delete_from_stores(key, _local)
+            def __delitem__(self, key: str) -> None: _delete_from_stores(key, _local)
+            def __contains__(self, key: str) -> bool: return key in _local
+            def __eq__(self, other: object) -> bool:
+                if isinstance(other, LocalProxy): return _local == other._local
```

Ruff (0.11.9)

Multiple statements on one line (colon)

(E701)